### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by ycal4269** [h-mdm/hmdm-server#68]
Hi guys,
fresh debian 11
fresh installation
and still getting an "cp: missing destination file operand after '/var/lib/tomcat9/webapps/ROOT.war'" when launcing the install script.
database's fine
what am I doing wrong?

**Comment by ycal4269** [comment]
indeed, i didn't see i didn't succeed in the mvn install step. So, I re-checked every step and I'm still getting an "ERROR: failed to deploy WAR file!"
So i checked catalina.out and i see 
```
[2022-12-20 12:29:07] [info] 2022-12-20 12:29:07 [INFO] liquibase.lockservice.StandardLockService : Successfully released change log lock
[2022-12-20 12:29:08] [info] 2022-12-20 12:29:08 [INFO] com.hmdm.persistence.ConfigurationDAO : Base URL: https://hmdm.fredon-aura.fr:8443
[2022-12-20 12:29:08] [info] [HMDM-INITIALIZER]: Application initialization was successful
[2022-12-20 12:29:08] [info] [HMDM-INITIALIZER]: Created a signal file for application initialization completion: /var/lib/tomcat9/work/
hmdm_install_flag
[2022-12-20 12:29:08] [info] 2022-12-20 12:29:08 [ERROR] com.hmdm.notification.guice.module.NotificationMqttTaskModule : Failed to create MQTT broker
service
[2022-12-20 12:29:08] [info] java.io.IOException: Failed to bind to server socket: mqtt://mydomain.com:31000 due to: java.net.BindException:
Cannot assign requested address (Bind failed)
```
*mydomain.com faked for privacy

Is it because port 31000 isn't publicly opened?
I understand I should open more ports than I am now. 8080, 8443, 31000 and ...? I'm missing other ports?


Thanks

**Comment by h-mdm** [comment]
Here's how to fix the bind issue: https://qa.h-mdm.com/1714/failed-to-create-mqtt-broker-service
List of open ports: https://qa.h-mdm.com/16781/which-ports-should-be-opened-to-make-headwind-mdm-working

Try to fix the issues, then run hmdm_install.sh and reinstall everything. Notice: run script as root, otherwise you may get the access denied issues.

**Comment by ycal4269** [comment]
Thank you, indeed opening ports solved everything.

I'm juste having now an issue with uploading files on devices (although already try https://qa.h-mdm.com/1925/) but everything else is working fine
